snippets/product-home.liquid

```diff
@@ -8,48 +8,71 @@
-    >
-      {{ block_settings.tag_text }}
-    </div>
-  {% endif %}
-    <img
-      loading='lazy'
-      class='main-image'
-      class='main-thumbnail'
-      src='{%- if images[0] %}{{images[0]}}{%- else %}{{ 'default_product.jpeg' | asset_url }}{% endif %}'
-      alt='{{ product_name }}'>
-  </div>
-  <div class='product-details'>
-    {%- for block in section.blocks -%}
-        {%- case block.type -%}
-          {%- when 'product_title' -%}
-            <h1 class='product-name'>{{ product.name }}</h1>
-            {%- render 'general-review' -%}
-          {%- when 'product_price' -%}
-            <div class='flex gap-2 items-center'>
-              <h2 class='product-price'>{{ product.price | money }}</h2>
-              {% if section.settings.show_compare_at_price %}
-                <h3>
-                  <del class="product-compare-price">{{ product.compare_at_price | money }}</del>
-                </h3>
+    <div class='product-details'>
+      {%- for block in section.blocks -%}
+          {%- case block.type -%} 
+            {%- when 'product_title' -%}
+              <div class="top-block">
+                {%- render 'general-review' -%}
+                <h1 class='product-name'>{{ product.name }}</h1>
+                  {% if block_settings.description %}
+                <div class="product-column-description">
+                    <p>{{ block_settings.description }}</p>
+                </div>
+            {% endif %}
+            </div>
+            <div class="bottom-block">
+              {%- when 'product_price' -%}
+              <div class='flex gap-3 items-center price-wrapper'>
+                {% if section.settings.show_compare_at_price %}
+                  <h3>
+                    <del class="product-compare-price">{{ product.compare_at_price | money }}</del>
+                  </h3>
+                {% endif %}
+                <h2 class='product-price'>{{ product.price | money }}</h2>
+              </div>
+            {%- when 'product_description' -%}
+              <section class='product-description '>{{ product.description }}</section>
+              {%- endcase -%}
+              {%- endfor -%}
+              <a href='{{ product.url }}' class='yc-btn buy-btn'>إتمام الشراء</a>
+              {% if block_settings.countdown_enabled %}
+                <div class="countdown-block">
+                  <span>{{ 'product_column.end' | t }} :</span>
+                  {%- render 'countdown', section_settings: block_settings -%}
+                </div>
               {% endif %}
+          </div>
+            
+      </div>
+        <div class='image-box'>
+          <div class="tag {{ block_settings.tag_position }}">
+            {% if block_settings.tag_text %} 
+              <div class="tag-text"
+                style="background: {{ block_settings.tag_color.hex }}; color: {{ block_settings.tag_text_color.hex }}"
```

Comment:
it would be better if you could include these style on top using `{% style %}` tag